src/main/java/eu/dissco/orchestration/backend/domain/MachineAnnotationService.java

```diff
@@ -26,4 +26,5 @@ public final class MachineAnnotationService {
   private String topicName;
   private int maxReplicas;
   private @NotBlank boolean batchingPermitted;
+  private final Integer timeToLive;
```

Comment:
Maybe we should say `@Positive` and check for a minimum or something.
I think now the user can put in -1000 or 0 and it would always timeout.
Maybe it should at least be an hour?

Reply:
added @Min annotation (it's inclusive)